ledger/ledger.go

```diff
@@ -72,7 +72,7 @@ type Ledger struct {
 
 	genesisAccounts map[basics.Address]basics.AccountData
 
-	genesisProto config.ConsensusParams
+	genesisProtoVersion protocol.ConsensusVersion
```

Comment:
`func (l *Ledger) OnlineTotals(rnd basics.Round) (basics.MicroAlgos, error) {` needs to use online account tracker + test